SnM/SnM_RegionPlaylist.cpp

```diff
@@ -485,13 +536,15 @@ void GetMonitoringInfo(WDL_FastString* _curNum, WDL_FastString* _cur,
 			// => best effort: find region by play pos
 			else
 			{
-				int id, idx = FindMarkerRegion(NULL, GetPlayPositionEx(NULL), SNM_REGION_MASK, &id);
-				if (id > 0)
-				{
-					char buf[64]="";
-					EnumMarkerRegionDesc(NULL, idx, buf, sizeof(buf), SNM_REGION_MASK, false, true, false);
-					_curNum->SetFormatted(16, "%d", GetMarkerRegionNumFromId(id));
-					_cur->Set(buf);
+				if (-1 != g_playCur) {
```

Comment:
Oops. This was not equivalent, but at least now when the playlist ends, the current region display immediately goes blank. Oh well.